### ISSUE TITLE
The compilation of ocaml-base-compiler failed

### ISSUE THREAD

**Issue by gregorst3** [ocaml/opam#3563]
**What did i do?**

`sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)`
`opam init --comp=4.06.1`

O.S: Tried with Debian 9 and Ubuntu 16.04, compilation of version 4.06.1 fail with a permission error.

```[ERROR] The compilation of ocaml-base-compiler failed at "/home/xxx/.opam/opam-init/hooks/sandbox.sh build ./configure -prefix
        /home/xxx/tezos/_opam -with-debug-runtime".

#=== ERROR while compiling ocaml-base-compiler.4.06.1 =========================#
# context     2.0.0 | linux/x86_64 |  | git+https://gitlab.com/tezos/opam-repository.git#4200cbebcae75562d79898a8e11e9f4dc237cc08
# path        ~/tezos/_opam/.opam-switch/build/ocaml-base-compiler.4.06.1
# command     ~/.opam/opam-init/hooks/sandbox.sh build ./configure -prefix /home/xxx/tezos/_opam -with-debug-runtime
# exit-code   1
# env-file    ~/.opam/log/ocaml-base-compiler-7014-05296d.env
# output-file ~/.opam/log/ocaml-base-compiler-7014-05296d.out
### output ###
# bwrap: Failed to make / slave: Permission denied




tail -f  /tmp/opam-root-6880/ocaml-base-compiler-6880-f0f09f.out
bwrap: Failed to make / slave: Permission denied

**Comment by gregorst3** [comment]
Fixed using opam init --comp=4.06.1 --disable-sandboxing

**Comment by rjbou** [comment]
You had several difficulties here: having your opam root in an unusual path and trying to use out of the box bwrap on ubuntu.
For the first one, instead of disabling completely sandboxing, you can just add in the `$OPAMROOT/opam-init/hooks/sandbox.sh` the directory `/build`, cf. #3434.
For the second one, you can referer to issue #3424 for bwrap & sandboxing.
